packages/lit-dev-content/site/docs/templates/directives.md

```diff
@@ -355,6 +400,178 @@ Explore `styleMap` more in the [playground](/playground/#sample=examples/directi
 
 ## Loops and conditionals
 
+### when
+
+<table>
+<thead><tr><th></th><th></th></tr></thead>
+<tbody>
+<tr>
+<td class="no-wrap-cell vcenter-cell">Import</td>
+<td class="wide-cell">
+
+```js
+import {when} from 'lit/directives/when.js';
+```
+
+</td>
+</tr>
+<tr>
+<td class="no-wrap-cell vcenter-cell">Signature</td>
+<td class="wide-cell">
+
+```ts
+choose<T, V>(
+  value: T,
+  cases: Array<[T, () => V]>,
+  defaultCase?: () => V
+)
+```
+</td>
+</tr>
+<tr>
+<td class="no-wrap-cell vcenter-cell">Usable location</td>
+<td class="wide-cell">
+
+Any
+
+</td>
+</tr>
+</tbody>
+</table>
+
+When `condition` is true, returns the result of calling `trueCase()`, else returns the result of calling `falseCase()` if `falseCase` is defined.
```

Comment:
In some of the sections here, the description comes after the import/signature/location table, while in others it comes before. Make it consistently before, I think, since that's how most of them were?

Reply:
Oops, this is just missing the description